apache/my-vhosts.conf

```diff
@@ -19,6 +19,8 @@
     ProxyPassReverse /save_to_sheet_from_form/ http://web:8000/save_to_sheet_from_form/
     ProxyPass /login http://web:8000/login
     ProxyPassReverse /login http://web:8000/login
+    ProxyPass /login http://web:8000/all
+    ProxyPassReverse /login http://web:8000/all
```

Comment:
For these two lines, you'll want to change "/login" to "/all", making this

```
    ProxyPass /all http://web:8000/all
    ProxyPassReverse /all http://web:8000/all
```

Reply:
Updated, although I'm not sure how to test this one.. How do you access the proxy when it's served by docker?